### PR TITLE
added db credential support for evelon module

### PR DESCRIPTION
**Pull request by Invincible2211**

### CODE REVIEW

File: src/main/java/de/yggdrasil/core/dal/strings/logging/DatabaseConnectionLogger.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package de.yggdrasil.core.dal.strings.logging;
+
+public interface DatabaseConnectionLogger {
```
Comment (Morazzer) [inline src/main/java/de/yggdrasil/core/dal/strings/logging/DatabaseConnectionLogger.java:3]: This doesn't fit the class naming scheme for the package

**Review by Morazzer — CHANGES_REQUESTED** [review]

File: src/main/java/de/yggdrasil/core/dal/DAL.java [hunk 2]
```diff
@@ -53,6 +61,28 @@ public <T extends DALReadRequest, V extends DALResponse> V read(T readRequest) {
         }
     }
 
+    /**
+     * Sets up the credentials for accessing the database used by the Evelon ORM module.
+     * This method reads the necessary information from the environment variables and configures the Evelon library with the provided credentials.
+     */
+    private void setupEvelonDBCredentials() {
+        Dotenv dotenv = Dotenv.configure().load();
```
Comment (Morazzer) [inline src/main/java/de/yggdrasil/core/dal/DAL.java:69]: Also is the extra library really required to read a file?

Comment (Invincible2211) [reply src/main/java/de/yggdrasil/core/dal/DAL.java:69]: In my opinion, it's unnecessary to write this function on my own when there is already a library for this exact purpose. To read this file, I could also use a file reader, but then I would have to check which value I read, or I would need to require a certain environment variable in a specific line (which is not good practice imo).

Comment (Morazzer) [reply src/main/java/de/yggdrasil/core/dal/DAL.java:69]: I mean you could use `java.util.Properties`

**Review by Morazzer — APPROVED** [review]

**Pull request merged**